frontend/public/components/pod.tsx

```diff
@@ -651,9 +654,53 @@ export const PodStatus: React.FC<PodStatusProps> = ({ pod }) => {
     (status === 'CrashLoopBackOff' || status === 'ErrImagePull' || status === 'ImagePullBackOff') &&
     containerStatusStateWaiting
   ) {
+    let footerLinks: React.ReactNode;
+    let headerTitle = '';
+    if (status === 'CrashLoopBackOff') {
+      headerTitle = t('public~Pod crash loop back-off');
+      const containers: ContainerSpec[] = pod.spec?.containers ?? [];
+      footerLinks = (
+        <div>
+          {t(
+            'public~CrashLoopBackOff indicates that the application within the container is failing to start properly.',
+          )}
+          <div className="co-divider-padding">
+            {t('public~To troubleshoot, view logs and events, then debug in terminal.')}
+          </div>
+          <Link to={`${resourcePath('Pod', pod.metadata.name, pod.metadata.namespace)}/logs`}>
+            {t('public~View logs')}
+          </Link>
+          &emsp;
+          <Link to={`${resourcePath('Pod', pod.metadata.name, pod.metadata.namespace)}/events`}>
+            {t('public~View events')}
+          </Link>
+          <Divider className="co-divider-padding" />
```

Comment:
Again, we should wrap the other element with a `p` instead of using `co-divider-padding`.

Reply:
The <p> is not providing the space I need:
![Screenshot 2021-09-02 at 20-36-25 Pods · OKD](https://user-images.githubusercontent.com/18728857/131942157-23ab4ebc-674b-4775-a598-06f1a4a7f637.png)
![Screenshot 2021-09-02 at 20-37-02 Pods · OKD](https://user-images.githubusercontent.com/18728857/131942171-072f6e9f-2250-4c97-8f7d-1754b386a597.png)

Reply:
Got this working with just the `p` element as you stated.